.circleci/config.yml

```diff
@@ -92,6 +92,12 @@ commands:
           name: 'Fail Fast'
           when: on_fail
           command: ./.circleci/fail_fast.sh
+  save_artifacts:
```

Comment:
nit: For clarity, since the two underlying steps start with `store` and not `save`, how about renaming this command to `store_test_output`?